tests/view/crudeapi/test_comment.py

```diff
@@ -0,0 +1,155 @@
+import datetime
+
+import pytest
+from werkzeug.exceptions import abort, NotFound
+
+from http import HTTPStatus
+from timeless.reservations.views import CommentView
+from timeless.reservations.views import Comment
+
+"""
+    Tests for CommentView
+    @todo #222:30min Finish CommentView implementation and CommentView tests. 
+     Methods for GET, POST, PUT and DELETE operation must be implemented, after the 
+     implementation remove the skip annotations from these tests.
+
+"""
+found_comment = {
+    "comment" : {
+        "comment_id" : 5,
+        "employee_id" : 10,
+        "body" : "This is a comment that exists",
+        "date" : "4000-01-01"
+    }
+}
+
+created_comment = {
+    "comment" : {
+        "employee_id" : 10,
+        "body" : "This is a comment that is newly created",
+        "date" : "4000-01-01"
+    }
+}
+
+
+class FakeComment(Comment):
```

Comment:
@paulodamaso let's remove `FakeComment` and use just `Comment` class in the tests? `get`, `post` and etc methods are useless here as well, since it's not a View / Controller. also since it inherits `Comment`, SQL-Alchemy will think it's a model as well and will try to create a table for that

Reply:
@vryazanov I don't like the idea. First of all this mean to be an unit test, not an integration test, so database is not supposed to be involved. Second here we are testing `CrudAPIView` methods, not `Comment` ones.  But I'll remove its inheritance from `Comment`